addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -651,11 +651,22 @@ function factory(dependencies) {
             const convertedData = this.env.models['mail.message'].convertData(data);
             const messageIds = this.env.models['mail.message'].all().map(message => message.id);
             const partnerRoot = this.env.messaging.partnerRoot;
-            this.env.models['mail.message'].create(Object.assign(convertedData, {
+            const message = this.env.models['mail.message'].create(Object.assign(convertedData, {
                 author: [['link', partnerRoot]],
                 id: (messageIds ? Math.max(...messageIds) : 0) + 0.01,
                 isTransient: true,
             }));
+            for (const channelId of data.channel_ids) {
+                const thread = this.env.models['mail.thread'].findFromIdentifyingData({
+                    id: channelId,
+                    model: 'mail.channel'
+                });
+                thread.threadViews.map(threadView =>
+                    threadView.addComponentHint('current-partner-just-posted-message', {
```

Comment:
I'm still not sure this is correct, maybe we should split the PR into what is obvious and what is not.

For the auto scroll, fixing it for normal messages as per task I linked earlier might be enough.

Reply:
This is task-2336251, I think it's best if we block the current PR until that one is done. Things will become clearer after.

Reply:
I'm taking it, I'll let you know when it's done and update this task spec accordingly.

Reply:
As I understood, task-2336251 is waiting for https://www.odoo.com/web#cids=1&id=2341740&menu_id=&model=project.task&view_type=form

Reply:
Yes, it's highlighting the bug you fixed of flickering new message so we can't merge it without the fix.

The current PR here (about commands) is not as much priority, focus on the task you just linked first.

Reply:
You should have a look as it's "ready" except a pending comment https://github.com/odoo/odoo/pull/57997